code/index.html

```diff
@@ -3,28 +3,121 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Project 3 - Business Site</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Work+Sans:wght@400;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <!-- Hero Section -->
+
+    <header class="hero-section">
+      <video class="hero-video" controls autoplay muted loop>
+        <source src="./hero-video.mp4" type="video/mp4" media="(max-width: 767px)">
+        <p>This browser doesn't support video...</p>
+      </video>
+      <section class="headline">
+        <h1 class="hero-title">MT Construction</h1>
+        <p class="hero-tagline">building more<br> with less</p>
+      </section>
+      </header>
 
     <!-- Signup form -->
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    <main class="main-area">
+      <section class="contact">
+
+        <form class="contact-form" action="http://httpbin.org/anything" method="POST">
+            
+            <legend class="contact-header">Please contact me</legend>
```

Comment:
Have you seen the `<fieldset>` element? 🤩

Reply:
Yes, I think I have (here I want to add a suitable emoji but I havn't figured out how yet ;)) I've checked up the fieldset element again, added it and removed the divs I had. It looks cleaner in the code file :) However, I want to got back later and improve the styling. Thanks a lot!